ruby/cross_service_examples/item-tracker/helpers/populate_table.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "yaml"
+require "aws-sdk-rdsdataservice"
+require_relative("../db_wrapper")
+
+client = Aws::RDSDataService::Client.new
+config = YAML.safe_load(File.open(File.join(File.dirname(__FILE__), "./../helpers", "config.yml")))
+wrapper = DBWrapper.new(config, client)
+
+10.times do
+  username = ["ltolstoy", "jsteinbeck", "jkerouac", "wkhalifa"].sample
```

Comment:
Which one of these is not like the other? 🤔

Reply:
Tolstoy obviously, the Russian!